src/vibe_llama/scaffold/samples/web_scraping.py

```diff
@@ -0,0 +1,83 @@
+WEB_SCRAPING_CODE = """
```

Comment:
python in a string seems a bit messy. I've been using the python lib copier (has both cli and programmatic access) for scaffolding things in llamactl. It can do fancy stuff with templating, upgrading, etc. I'd recommend looking into it. You could probably just use it similarly as you do here with the python code in a directory.

Keep in mind that we should also be maintaining these closely, ensuring that they are up to date and functional as workflows etc. changes. Having them as isolated projects with strict versions allows us to do some static checking to make sure that they are up to snuff.